arm/Microsoft.Compute/virtualMachines/.parameters/windows.parameters.json

```diff
@@ -12,7 +12,7 @@
             "value": {
```

Comment:
Cannot create a suggestion for code you didn't change - hence will do it here:

```bicep
"name": {
  "value": "<<namePrefix>>-vm-win-001"
}
```

Reply:
we changed to 01 after allowing namePrefix to 5 chars (thus carml) due to the limitation of max 15 chars for win vm names